include/pcl_conversions/pcl_conversions.h

```diff
@@ -107,31 +107,36 @@ namespace pcl_conversions {
   /** PCLHeader <=> Header **/
 
   inline
-  void fromPCL(const pcl::PCLHeader &pcl_header, std_msgs::Header &header)
+  void fromPCL(const pcl::PCLHeader &pcl_header, std_msgs::msg::Header &header)
   {
     fromPCL(pcl_header.stamp, header.stamp);
-    header.seq = pcl_header.seq;
+    // TODO(clalancette): Seq doesn't exist in the ROS2 header
+    // anymore.  Do we need to do anything here?
+    //header.seq = pcl_header.seq;
     header.frame_id = pcl_header.frame_id;
   }
 
   inline
-  void toPCL(const std_msgs::Header &header, pcl::PCLHeader &pcl_header)
+  void toPCL(const std_msgs::msg::Header &header, pcl::PCLHeader &pcl_header)
   {
     toPCL(header.stamp, pcl_header.stamp);
-    pcl_header.seq = header.seq;
+    // TODO(clalancette): Seq doesn't exist in the ROS2 header
+    // anymore.  Do we need to do anything here?
+    //pcl_header.seq = header.seq;
+    pcl_header.seq = 0;
```

Comment:
There should be some way to get the sequence number from the system in the future (part of the rationale was that the middleware should be able to answer this question for all messages anyways and either per publisher or per received), but for now leaving it 0 is fine by me.